pug/test_viewResult.pug

```diff
@@ -37,6 +37,21 @@ block extrascripts
       form.nosubmit = true;
       form.submission = testdata;
     }
+  
+  script(type = 'text/javascript').
+    
+    function GetWireTension()
+    {
+      var layer = document.getElementById("wireLayer").value;
+      var side = document.getElementById("apaSide").value;
+      var number = document.getElementById("wireNumber").value;
+      
+      var keyName = "wires." + layer + "." + side + "[" + number + "]";
```

Comment:
Just as an FYI, you can use string interpolation instead of concatenation:
```
const keyName = `wires.${layer}.${side}[${number}]`;
```

Reply:
I actually tried this exactly as you've written, and it didn't work for some reason.

Reply:
If the code gets transpiled, the PUG transpiler might not support ESNext, in which case we might just need to update it.  Natively, however, all modern browsers support this feature.

Reply:
To be fair, I don't actually know why it wasn't working ... I'm aware that string interpolation should work with my browser, so I knew it wasn't that which was the issue.  I'll give it another go ... might have been something else on the page that was breaking things (I was playing with other things on it at the same time).